week-3/Homework/mandatory/5-journey-planner.js

```diff
@@ -84,7 +86,11 @@ function getTransportModes() { }
 
   Hint: Use the corresponding array method to decide if an element is member of an array.
 */
-function isAccessibleByTransportMode() { }
+function isAccessibleByTransportMode(arr, str) {
+  if (arr.includes(str)) {
+    return true;
```

Comment:
Do you need to have if the condition
what are the possible results of include?

Reply:
includes() method already does the checking and return true or false, so I did not need to use if to check again. return arr.includes(str) did the job.